docs/setup-cluster/deploy-cluster/installation-options.rst

```diff
@@ -1,20 +1,135 @@
-#############
- Basic Setup
-#############
+:orphan:
 
-Your training environment can be a local development machine, an on-premise GPU cluster, or cloud
-resources. To set up your training environment, follow the steps below.
+.. _installation-options:
 
-**************************************************
- Step 1 - Set ``DET_MASTER`` Environment Variable
-**************************************************
+######################
+ Installation Options
+######################
+
+In this guide, you'll find information about basic and advanced cluster configuration options, as
+well as additional installation options such as setting up clients, internet access, and firewall
+rules.
+
+.. _cluster-configuration:
+
+***********************
+ Configure the Cluster
+***********************
+
+Common configuration reference: :doc:`/reference/deploy/config/common-config-options`
+
+Master configuration reference: :doc:`/reference/deploy/config/master-config-reference`
+
+Agent configuration reference: :doc:`/reference/deploy/config/agent-config-reference`
+
+Basic Configuration
+===================
+
+The behavior of the master and agent can be controlled by setting configuration variables; this can
```

Comment:
"Configuration variables" is not a term that I would normally ever hear.  It sounds like some sort of Proper Noun in our system.  Better to omit that phrase:

> The master and agent can each be configured with a configuration file, environment variables, or command-line options.

Reply:
resolved